File: app/src/main/java/jogja/telkom/csrassistant/Tagihan/TagihanActivity.java
package jogja.telkom.csrassistant.Tagihan;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import jogja.telkom.csrassistant.R;

public class TagihanActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tagihan);

        Button tidakSesuai = findViewById(R.id.tidak_sesuai);
        Button tetapDitagih = findViewById(R.id.tetap_ditagih);

        tidakSesuai.setOnClickListener(v ->
                startActivity(new Intent(TagihanActivity.this, TidakSesuaiTagihanActivity.class)));

        tetapDitagih.setOnClickListener(v ->
                startActivity(new Intent(TagihanActivity.this, TetapDitagihTagihanActivity.class)));
    }
}

File: app/src/main/java/jogja/telkom/csrassistant/Tagihan/TagihanActivity.kt
package jogja.telkom.csrassistant.Tagihan

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import jogja.telkom.csrassistant.R

class TagihanActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tagihan)

        val tidakSesuai = findViewById<Button>(R.id.tidak_sesuai)
        val tetapDitagih = findViewById<Button>(R.id.tetap_ditagih)

        tidakSesuai.setOnClickListener { v -> startActivity(Intent(this@TagihanActivity, TidakSesuaiTagihanActivity::class.java)) }

        tetapDitagih.setOnClickListener { v -> startActivity(Intent(this@TagihanActivity, TetapDitagihTagihanActivity::class.java)) }
    }
}
